src/Beberlei/Metrics/Collector/GaugeableCollector.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Beberlei Metrics
+ *
+ * LICENSE
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this package in the file LICENSE.txt.
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to kontakt@beberlei.de so I can send you a copy immediately.
+ */
+
+namespace Beberlei\Metrics\Collector;
+
+interface GaugeableCollector extends Collector
```

Comment:
@lyrixx but currently, implementing  GaugeableCollector without Collector does not make sense, as ``flush`` is part of Collector. If you don't extend it, you should move ``flush`` to a base interface extended by both GaugeableCollector and Collector.

And implementing multiple interfaces extending from the same interface is perfectly fine. Asking you twice to fulfil the ``Collector`` contract is not an issue. It is just asking you to fulfil it

the case where it would actually cause issue is not when introducing Bucket (which would also be a separate interface) but when introducing a new collector which supports gauge() but not one of the basic methods (as it would not be able to implement Collector). But the issue there would be to have all basic methods in the same interface, not that GaugeableCollector extends it.